cumulusci/tasks/salesforce/activate_flow.py

```diff
@@ -12,43 +12,58 @@ class ActivateFlow(BaseSalesforceApiTask):
         "developer_names": {
             "description": "List of DeveloperNames to query in SOQL",
             "required": True,
-        }
+        },
+        "status": {
+            "description": "Flag to check whether to activate or deactivate the flow",
+            "required": False,
+        },
     }
 
     def _init_options(self, kwargs):
         super(ActivateFlow, self)._init_options(kwargs)
         self.options["developer_names"] = process_list_arg(
             self.options.get("developer_names")
         )
-        self.api_version = "43.0"
+        self.api_version = "58.0"
         if not self.options["developer_names"]:
             raise TaskOptionsError(
                 "Error you are missing developer_names definition in your task cumulusci.yml file. Please pass in developer_names for your task configuration or use -o to developer_names as a commandline argument"
             )
 
     def _run_task(self):
-        self.logger.info(
-            f"Activating the following Flows: {self.options['developer_names']}"
-        )
+        if self.options["status"]:
```

Comment:
@lakshmi2506 
Consider the test case that status was not entered in the options:
```
create_task(
            ActivateFlow,
            {
                "developer_names": [
                    "Auto_Populate_Date_And_Name_On_Program_Engagement",
                    "ape",
                ],
            },
        )
```
Would this code still pass or get a key error, since status is not a task option.

To fix this I recommend adding 

```
self.options["status"] = process_list_arg(
            self.options.get("status"), False
        )
        
```
to the _init_options() section.